http/centos-7.3/ks-openstack.cfg

```diff
@@ -0,0 +1,86 @@
+install
```

Comment:
So following the convention that we have started using in this template, can you please rename this file to `http/centos-7.3/ks-x86_64.cfg` ?